### DISCUSSION TITLE
Populate users name with Google Oauth data

### DISCUSSION THREAD

**Discussion by FranckPoingt** [pocketbase/pocketbase#4421] [Q&A] [answered]
Hi, 
When authenticating with Google Oauth, how can I populate the `name` field of my table with the one provided by google? 

Is it possible to do something like: 
```ts
await locals.pb.collection('users').authWithOAuth2Code(
			provider.name,
			code,
			provider.codeVerifier,
			PUBLIC_REDIRECT_URL + provider.name
			// {
			// 	name:  something.meta.name
			// }
		);
```
Or do I need to make 2 calls, first to sign in then to update the record? 

Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
At the moment you'll have to send an additional update call (or attach to the [OnRecordAfterAuthWithOAuth2Request](https://pocketbase.io/docs/go-event-hooks/#onrecordafterauthwithoauth2request) hook and update `e.Record`).

**Reply by FranckPoingt** [reply]
Sweet, I'll stick to the additional update call for now but thanks for pointing out the doc to the hook. 
Thank you for this awesome product.
